src/main/java/com/ericsson/eiffel/remrem/semantics/clone/CloneEiffelRepository.java

```diff
@@ -0,0 +1,42 @@
+package com.ericsson.eiffel.remrem.semantics.clone;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.Git;
+
+import com.ericsson.eiffel.remrem.semantics.eiffelSchemas.EiffelRepoJsonSchema;
+import com.ericsson.eiffel.remrem.semantics.eiffelSchemas.EiffelConstants;
+
+/**
+ * This class is used to Clone the Eiffel Repo from github
+ * 
+ * @author xdurvak
+ */
+
+public class CloneEiffelRepository {
```

Comment:
This is a class that is hard to name. I wonder if PrepareLocalEiffelSchemas would be a better name. I think the most important part is that we modify the schemas. Cloning is not the focus here.